fix(button): set title on button when is set on host (VIV-1249) (#1294)
Vonage/vivid-3: libs/components/src/lib/button/README.md

<vwc-button appearance="outlined" icon="check-line" label="outlined with icon" pending></vwc-button>
```

## Accessibility 

> When setting an `aria-label`, `aria-labelledby` or `title`, mind that `aria-label`, `aria-labelledby` are better for a11y.  
> When both `aria-label` and `title` are set, `aria-label` text will be read by the assistive technologies and not `title`.

Reviewer: ```suggestion
When setting an `aria-label`, `aria-labelledby` or 'title', keep in mind that `aria-label`, `aria-labelledby` are better for A11Y.  
The assistive technology will read the `aria-label` text rather than the `title` if both are set.
```
